app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuEnhancedTrackingProtectionExceptionsRobot.kt

```diff
@@ -6,11 +6,16 @@
 
```

Comment:
Add here the header... /* This Source Co.....

Reply:
ok